bftengine/src/bftengine/BFTEngine.cpp

```diff
@@ -138,7 +139,7 @@ IReplica *IReplica::createNewReplica(ReplicaConfig *replicaConfig,
     ((PersistentStorageImp *)persistentStoragePtr.get())->init(move(metadataStoragePtr));
   }
 
-  auto *replicaInternal = new ReplicaInternal();
+  auto replicaInternal = new ReplicaInternal();
```

Comment:
As for me, "auto *" helps to understand that an object of type pointer is created. I would leave it as is.